hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowParquetWriteSupport.java

```diff
@@ -118,7 +118,7 @@ public class HoodieRowParquetWriteSupport extends WriteSupport<InternalRow> {
   private final Function1<Object, Object> timestampRebaseFunction = DataSourceUtils.createTimestampRebaseFuncInWrite(datetimeRebaseMode, "Parquet");
   private final boolean writeLegacyListFormat;
   private final ValueWriter[] rootFieldWriters;
-  private final Schema avroSchema;
+  private final HoodieSchema hoodieSchema;
```

Comment:
nitpick: let's update `hoodieSchema` variable names to simply `schema`?

Reply:
Done